tutorials/preprocessing/plot_15_handling_bad_channels.py

```diff
@@ -0,0 +1,254 @@
+data:
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+
+###############################################################################
+# Marking bad channels
+# ^^^^^^^^^^^^^^^^^^^^
+#
+# Sometimes individual channels malfunction and provide data that is too noisy
+# to be usable. MNE-Python makes it easy to remove bad channels from the
+# analysis stream without actually deleting the data in those channels. It does
+# this by
+# keeping track of the bad channel indices in a list and looking at that list
+# when doing analysis or plotting tasks. The list of bad channels is stored in
+# the `'bads'` field of the :class:`~mne.Info` object that is attached to
+# :class:`~mne.io.Raw`, :class:`~mne.Epochs`, and :class:`~mne.Evoked` objects.
+
+print(raw.info['bads'])
+
+###############################################################################
+# Here you can see that the :file:`.fif` file we loaded from disk must have
+# been keeping track of channels marked as "bad" — which is good news, because
+# it means any changes we make to the list of bad channels will be preserved if
+# we save our data at intermediate stages and re-load it later. Since we saw
+# above that ``EEG 053`` is one of the bad channels, let's look at it alongside
+# some other EEG channels to see what's bad about it. We can do this using the
+# standard :meth:`~mne.io.Raw.plot` method, and instead of listing the channel
+# names one by one (``['EEG 050', 'EEG 051', ...]``) we'll use a `regular
+# expression`_ to pick all the EEG channels between 050 and 059 with the
+# :func:`~mne.pick_channels_regexp` function (the ``.`` is a wildcard
+# character):
+
+picks = mne.pick_channels_regexp(raw.ch_names, regexp='EEG 05.')
+raw.plot(order=picks, n_channels=len(picks))
+
+###############################################################################
+# We can do the same thing for the bad MEG channel (``MEG 2443``). Since we
+# know that Neuromag systems (like the one used to record the example data) use
+# the last digit of the MEG channel number to indicate sensor type, here our
+# `regular expression`_ will pick all the channels that start with 2 and end
+# with 3:
+
+picks = mne.pick_channels_regexp(raw.ch_names, regexp='MEG 2..3')
+raw.plot(order=picks, n_channels=len(picks))
+
+###############################################################################
+# Notice first of all that the channels marked as "bad" are plotted in a light
+# gray color in a layer behind the other channels, to make it easy to
+# distinguish them from "good" channels. The plots make it clear that ``EEG
+# 053`` is not picking up scalp potentials at all, and ``MEG 2443`` looks like
+# it's picking up a lot *more* than its neighbors — its signal is a few orders
```

Comment:
reading the sentence as it is, I would comprehend "picking up a lot more" as "picking up a lot more scalp potentials", which is not true for MEG 2443.